src/providers.js

```diff
@@ -58,6 +58,20 @@ class Providers {
     this.syncQueue = new Queue({ concurrency: 1 })
   }
 
+  /**
+   * Start the provider cleanup service
+   */
+  start () {
+    if (this._cleaner) {
+      clearInterval(this._cleaner)
+    }
```

Comment:
The check isn't necessary in `stop` and `start` since clearInterval ignores null/undefined values.